Bottom: b1f20c92fe61f3ed09dd893b46eb0ec181bee9b4
Top:    ee7ef8e12d3d6bf451c61a7af6c2281a784a5f96
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-12 11:02:51 +0800

Refresh of vhost_net-introduce

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index de68598..d0a3c30 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -481,8 +481,8 @@ static int init_iov_iter(struct vhost_virtqueue *vq, struct iov_iter *iter,
 
 static bool vhost_exceeds_weight(int pkts, int total_len)
 {
-	return total_len >= VHOST_NET_WEIGHT ||
-	       pkts >= VHOST_NET_PKT_WEIGHT;
+	return unlikely(total_len >= VHOST_NET_WEIGHT) ||
+	       unlikely(pkts >= VHOST_NET_PKT_WEIGHT);
 }
 
 /* Expects to be always run from workqueue - which acts as
